tools/shared.py

```diff
@@ -57,9 +57,9 @@
 # Minimum node version required to run the emscripten compiler.  This is
 # distinct from the minimum version required to execute the generated code
 # (settings.MIN_NODE_VERSION).
-# This version currently matches the node version that we ship with emsdk
-# which means that we can say for sure that this version is well supported.
-MINIMUM_NODE_VERSION = (16, 20, 0)
+# This is currently set ot v18 since this is the version of node available
+# in debian/stable (bookworm).
```

Comment:
Ah, I was going to ask "why 18", but yeah, debian stable is a reasonable reason.

Reply:
Its also the version in /usr/bin/node on our google machines.